src/js/background/context-menu.js

```diff
@@ -281,20 +288,25 @@ const relayContextMenus = {
       const { hostname } = new URL(url);
       return hostname;
     },  
-    getMostRecentAliases: (array, domain)=> {
-      // Flipped to match the same order as the dashboard
-      array.reverse();
+    getMostRecentAliases: (array, domain, shouldFlipArray)=> {
+      // Flipped to match the same order as the dashboard if synced from the server
+      if (shouldFlipArray) {
```

Comment:
I generally prefer turning optional parameters into named parameters. That way, if we have to add more parameteres in the future, you don't have to pass in a bunch of `undefined`s.

I also prefer using named parameters for boolean arguments (which often are optional), just because it's clearer to people reading code at the call site to determine what that parameter does.

```suggestion
    getMostRecentAliases: (array, domain, options = {})=> {
      // Flipped to match the same order as the dashboard if synced from the server
      if (options.shouldFlipArray) {
```

So this would be called as:

```js
getMostRecentAliases(aliases, currentWebsite, { shouldAliasOrderBeReversed: true })`.
```

Likewise for the other instance of this parameter.

Reply:
I dig it! I was already using an options argument on the parent function. Good call out on using it in all the spots!

Reply:
Revised in 54a57e0b85ff4b99e59537b9d7a319f25cb07bb2